LICENSE.md

```diff
@@ -93,21 +90,21 @@ A Derivative must contain changes which a reasonably informed person would consi
 The User may distribute a Derivative if the following conditions are met:
  - The Derivative does not generate revenue.
  - The Derivative provides credit to the Author.
- - The Derivative complies with the Mod license.
+ - The Derivative, as a whole, is licensed under this license for anyone who comes into possession of a copy.
```

Comment:
"complies with the license" was too vague. And I don't quite understand what you mean with that.
I took this from the GNU GPL